cvat/apps/iam/rules/webhooks_test.gen.rego

```diff
@@ -0,0 +1,3002 @@
+    not allow with input as {"scope": "create@organization", "auth": {"user": {"id": 81, "privilege": "user"}, "organization": {"id": 155, "owner": {"id": 241}, "user": {"role": null}}}, "resource": {"id": null, "owner": {"id": 185}, "organization": {"id": 227}, "num_resources": 0}}
+}
+
+test_scope_CREATE_IN_ORGANIZATION_context_ORGANIZATION_ownership_NONE_privilege_USER_membership_NONE_resource_num_resources_11_same_org_FALSE {
+    not allow with input as {"scope": "create@organization", "auth": {"user": {"id": 1, "privilege": "user"}, "organization": {"id": 121, "owner": {"id": 235}, "user": {"role": null}}}, "resource": {"id": null, "owner": {"id": 114}, "organization": {"id": 230}, "num_resources": 11}}
+}
+
+test_scope_CREATE_IN_ORGANIZATION_context_ORGANIZATION_ownership_NONE_privilege_WORKER_membership_OWNER_resource_num_resources_0_same_org_TRUE {
+    allow with input as {"scope": "create@organization", "auth": {"user": {"id": 1, "privilege": "worker"}, "organization": {"id": 105, "owner": {"id": 1}, "user": {"role": "owner"}}}, "resource": {"id": null, "owner": {"id": 174}, "organization": {"id": 105}, "num_resources": 0}}
+}
+
+test_scope_CREATE_IN_ORGANIZATION_context_ORGANIZATION_ownership_NONE_privilege_WORKER_membership_OWNER_resource_num_resources_11_same_org_TRUE {
+    not allow with input as {"scope": "create@organization", "auth": {"user": {"id": 88, "privilege": "worker"}, "organization": {"id": 160, "owner": {"id": 88}, "user": {"role": "owner"}}}, "resource": {"id": null, "owner": {"id": 150}, "organization": {"id": 160}, "num_resources": 11}}
+}
+
+test_scope_CREATE_IN_ORGANIZATION_context_ORGANIZATION_ownership_NONE_privilege_WORKER_membership_OWNER_resource_num_resources_0_same_org_FALSE {
+    not allow with input as {"scope": "create@organization", "auth": {"user": {"id": 11, "privilege": "worker"}, "organization": {"id": 131, "owner": {"id": 11}, "user": {"role": "owner"}}}, "resource": {"id": null, "owner": {"id": 176}, "organization": {"id": 219}, "num_resources": 0}}
+}
+
+test_scope_CREATE_IN_ORGANIZATION_context_ORGANIZATION_ownership_NONE_privilege_WORKER_membership_OWNER_resource_num_resources_11_same_org_FALSE {
+    not allow with input as {"scope": "create@organization", "auth": {"user": {"id": 63, "privilege": "worker"}, "organization": {"id": 111, "owner": {"id": 63}, "user": {"role": "owner"}}}, "resource": {"id": null, "owner": {"id": 147}, "organization": {"id": 298}, "num_resources": 11}}
+}
+
+test_scope_CREATE_IN_ORGANIZATION_context_ORGANIZATION_ownership_NONE_privilege_WORKER_membership_MAINTAINER_resource_num_resources_0_same_org_TRUE {
+    allow with input as {"scope": "create@organization", "auth": {"user": {"id": 7, "privilege": "worker"}, "organization": {"id": 107, "owner": {"id": 245}, "user": {"role": "maintainer"}}}, "resource": {"id": null, "owner": {"id": 179}
```

Comment:
I don't see a script that was used to generate the file. Please look at other rego files. How to reproduce the file?